package.json

```
@@ -76,7 +76,7 @@
     }
   },
   "optionalDependencies": {
-    "@mattrglobal/node-bbs-signatures": "0.12.0"
+    "@mattrglobal/node-bbs-signatures": "^0.13.0"
```

Because this is pre 1.0 we prefer to lock the version rather than leaving it floating

Please update to
```suggestion
    "@mattrglobal/node-bbs-signatures": "0.13.0"
```